feat: add my reservations list

Valenreynoso17 opened the pull request:
que sea lo que dios quiera

Valenreynoso17 committed:
feat: add my reservations list

ChortJulio commented on src/main/java/com/barapp/web/views/MisReservasView.java:
```diff
@@ -0,0 +1,642 @@
+//        reservasPendientesList.add(new Reserva(EstadoReserva.PENDIENTE, 2, LocalDate.of(2024, 10, 22), new Horario(LocalTime.of(21, 0), TipoComida.CENA), new UsuarioApp("Saraceli", "Arina", "", "", null), new Restaurante(), "", TipoComida.CENA, ""));
+//        reservasPendientesList.add(new Reserva(EstadoReserva.PENDIENTE, 5, LocalDate.of(2024, 9, 22), new Horario(LocalTime.of(9, 0), TipoComida.DESAYUNO), new UsuarioApp("Julio", "Quarin", "", "", null), new Restaurante(), "", TipoComida.DESAYUNO, ""));
+//        reservasPendientesList.add(new Reserva(EstadoReserva.PENDIENTE, 3, LocalDate.of(2024, 11, 15), new Horario(LocalTime.of(16, 5), TipoComida.MERIENDA), new UsuarioApp("Federico", "Reynoso", "", "", null), new Restaurante(), "", TipoComida.MERIENDA, ""));
+        reservasPendientesList.add(new Reserva(EstadoReserva.PENDIENTE, 10, LocalDate.now(), new Horario(LocalTime.now().plusMinutes(10), TipoComida.ALMUERZO), new UsuarioApp("Sagustina", "Ander", "", "", null), new Restaurante(), "", TipoComida.ALMUERZO, ""));
```
Los 3 constructores de los usuarios ahora tiran error, les falta un último atributo agregado hace 2 semanas por Fede llamado  private Set<String> fcmTokens;
Probé con null y HashSet vacío y no pude

ChortJulio commented on src/main/java/com/barapp/web/views/MisReservasView.java:
```diff
@@ -0,0 +1,642 @@
+//        reservasPasadasList.add(new Reserva(EstadoReserva.CANCELADA_USUARIO, 3, LocalDate.of(2024, 6, 19), new Horario(LocalTime.of(13, 30), TipoComida.ALMUERZO), new UsuarioApp("Sagustina", "Ander", "", "", null), new Restaurante(), "", TipoComida.ALMUERZO, ""));
+//        reservasPasadasList.add(new Reserva(EstadoReserva.CONCRETADA, 12, LocalDate.of(2024, 2, 11), new Horario(LocalTime.of(16, 5), TipoComida.MERIENDA), new UsuarioApp("Federico", "Reynoso", "", "", null), new Restaurante(), "", TipoComida.MERIENDA, ""));
+//        reservasPasadasList.add(new Reserva(EstadoReserva.CONCRETADA, 5, LocalDate.of(2024, 1, 27), new Horario(LocalTime.of(16, 5), TipoComida.MERIENDA), new UsuarioApp("Marcar no cumplida en 20m", "", "", "", null), new Restaurante(), "", TipoComida.MERIENDA, ""));
+//        reservasPasadasList.add(new Reserva(EstadoReserva.CANCELADA_BAR, 9, LocalDate.of(2024, 4, 22), new Horario(LocalTime.of(16, 5), TipoComida.MERIENDA), new UsuarioApp("Marcar no cumplida", "", "", "", null), new Restaurante(), "", TipoComida.MERIENDA, ""));
```
No banco todo este código comentado im sorry

ChortJulio reviewed (dismissed):
Sólo vi 1 archivo (MisReservasView.java)

FedericoQuarin committed:
fix: fix error on login

Se corrigio un error al logearse con cualquier usuario. Se estaba buscando mediante el import el @fontsource/sansita-swashed/400.css innecesariamente. Parece que el error aparecio recien ahora por el cambio de version.

ChortJulio commented:
Ahora hay 2 conflictos y no hay trazabilidad (nombre de los commits no tienen el código BA-26 delante del nombre)

FedericoQuarin reviewed (approved):
I think this is the best PR I've ever seen in my life
